docs/enrollRewardProgram.md

```diff
@@ -1,19 +1,19 @@
-`symbol-bootstrap enrolRewardProgram`
-=====================================
+`symbol-bootstrap enrollRewardProgram`
+======================================
 
-It enrols the nodes in the rewards program by announcing the enrol transaction to the registration address.  You can also use this command to update the program registration when you change the node public key or server host.
+It enrols the nodes in the rewards program by announcing the enroll transaction to the registration address.  You can also use this command to update the program registration when you change the agent keys (changing the agent-ca-csr) or server host.
 
 Currently, the only program that can be enrolled post-launch is 'SuperNode'.
```

Comment:
We should discuss how the 'Ecosystem' and 'EarlyAdoption' node owners will enroll with the new agentCaCsr? the only place is Nis1 for now, I guess.

Reply:
One Idea is we populate the initial node entry in the collection with an empty certificate and we tell node admins to send the eroll message. Early and ecosystem can just update the node entry, not create